replay-tool-it/pom.xml

```diff
@@ -10,12 +10,12 @@
     <modelVersion>4.0.0</modelVersion>
     <packaging>pom</packaging>
     <modules>
-        <module>framework-event-listener</module>
-        <module>framework-tools-it</module>
+        <module>it-example-listener</module>
+        <module>integration-test</module>
```

Comment:
Given that the groupId is very generic (uk.gov.justice), I think these submodule names really need "framework-tools" at the beginning, otherwise we'll have a GAV of uk.gov.justice:integration-test:2.1.0 which will mean nothing to anyone.

Reply:
agreed